cuckoo_time_translator_algorithms/include/cuckoo_time_translator/AbstractAssert.h

```diff
@@ -3,6 +3,15 @@
 
 #include <console_bridge/console.h>
 
+#ifdef DEFINE_LOGGING_MACROS
+
+#define logError(fmt, ...)  CONSOLE_BRIDGE_logError(fmt, ##__VA_ARGS__) 
```

Comment:
I would generate compatibility macros that avoid collision with potential client code, i.e. `CUCKOO_TIME_logError` instead of `logError`. Of course these macros would require a definition in both cases (legacy and current).